### PR TITLE
Updated web common to 2.0.26

### PR DESCRIPTION
**Pull request by candrews55**

### CODE REVIEW

File: Gateway/Crossroads.Utilities.Test/Services/AppSettingsTypeConverterTest.cs [hunk 1]
```diff
@@ -1,49 +1,40 @@
...
-using System.Globalization;
-using Crossroads.Web.Common;
-using Crossroads.Web.Common.Configuration;
-
-namespace Crossroads.Utilities.Test.Services
+﻿namespace Crossroads.Utilities.Test.Services
 {
     class AppSettingsTypeConverterTest
     {
-        private AppSettingsTypeConverter fixture;
```
Comment (jpc11623) [inline Gateway/Crossroads.Utilities.Test/Services/AppSettingsTypeConverterTest.cs]: Why are these tests commented out? Is this code not being used now?

File: Gateway/MinistryPlatform.Translation.Test/Services/GroupParticipantRepositoryTest.cs [hunk 2]
```diff
@@ -130,9 +127,9 @@ public void GetAllGroupNamesLeadByParticipantShouldTransfomGroupParticipantIntoG
             _apiUserRepository.Setup(mocked => mocked.GetDefaultApiClientToken()).Returns("yeah!");
             _ministryPlatformRestRepository.Setup(mocked => mocked.UsingAuthenticationToken("yeah!")).Returns(_ministryPlatformRestRepository.Object);
             _ministryPlatformRestRepository.Setup(mocked => mocked.Search<MpGroupParticipant>(
-                It.Is<string>(searchString => MatchesDate(searchString, search)), 
+                It.Is<string>(searchString => MatchesDate(searchString, search)),
                 It.IsAny<string>(),
-                (string)null,
```
Comment (jpc11623) [inline Gateway/MinistryPlatform.Translation.Test/Services/GroupParticipantRepositoryTest.cs]: I am wondering about removing all these casts - was this due to errors being thrown in the compiler?

Comment (candrews55) [reply Gateway/MinistryPlatform.Translation.Test/Services/GroupParticipantRepositoryTest.cs]: Nope. I am thinking resharper may have done that. I can add them back in.

File: Gateway/Crossroads.Utilities.Test/Services/AppSettingsTypeConverterTest.cs (hunk 1 above)
Comment (candrews55) [reply Gateway/Crossroads.Utilities.Test/Services/AppSettingsTypeConverterTest.cs]: They look to be testing things in web common that no longer exist. I am going to reach out to Dillon to see if my assumption is true. I will remove them if they are not needed.

**Pull request merged**